src/recap.c

```diff
@@ -23,63 +23,38 @@
 #include <inttypes.h>
 #include <pcap.h>
 
+#include <eloop.h>
+
 #include "version.h"
 #include "utils/os.h"
 #include "capture/middlewares/header_middleware/sqlite_header.h"
 #include "capture/middlewares/header_middleware/packet_decoder.h"
 #include "capture/middlewares/header_middleware/packet_queue.h"
 #include "capture/middlewares/protobuf_middleware/protobuf_middleware.h"
+#include "capture/capture_service.h"
```

Comment:
@mereacre, before you merge this, don't merge https://github.com/nqminds/edgesec/pull/116, otherwise it will cause merge conflicts (it's easy to autoformat that PR to fix the merge conflicts)

Reply:
ok. I will merge this one first and then #116.